src/kubernetes_api_objects/config_map.rs

```diff
@@ -231,6 +231,14 @@ impl ResourceView for ConfigMapView {
     proof fn from_dynamic_preserves_metadata() {}
 
     proof fn from_dynamic_preserves_kind() {}
+
+    open spec fn rule(obj: DynamicObjectView) -> bool {
+        true
+    }
+
+    open spec fn transition_rule(new_cr: DynamicObjectView, old_cr: DynamicObjectView) -> bool {
```

Comment:
I think you can just rename the two arguments to `new_obj` and `old_obj` for all types